azure-pipelines.yml

```diff
@@ -24,11 +24,13 @@ steps:
     repository: '963234657927.dkr.ecr.us-east-1.amazonaws.com/ni-grafana'
```

Comment:
I didn't expect to see changes to this file.

Reply:
Yeah, neither. I believe it's because I branched off of ni/current. Just need to merge in real quick.